warehouse/query-core/src/main/java/datawave/query/config/ShardQueryConfiguration.java

```diff
@@ -3156,4 +3167,20 @@ protected Object readResolve() throws ObjectStreamException {
         this.fstCount = new AtomicInteger(0);
         return this;
     }
+
+    public boolean isUseQueryTreeScanHintRules() {
+        return useQueryTreeScanHintRules;
+    }
+
+    public void setUseQueryTreeScanHintRules(boolean useQueryTreeScanHintRules) {
```

Comment:
We need to add delegate methods to the ShardQueryLogic so that we can actually configure there.

Reply:
now how long its been since I had a query update is showing....